test/e2e/framework/assertions.go

```diff
@@ -105,17 +104,17 @@ func (f *Framework) AssertResourceEventuallyExists(name, namespace string, resou
 func (f *Framework) AssertStatefulsetReady(name, namespace string, fns ...OptionFn) func(t *testing.T) {
 	option := AssertOption{
 		PollInterval: 5 * time.Second,
-		WaitTimeout:  CustomForeverTestTimeout,
+		WaitTimeout:  DefaultTestTimeout,
 	}
 	for _, fn := range fns {
 		fn(&option)
 	}
 	return func(t *testing.T) {
 		key := types.NamespacedName{Name: name, Namespace: namespace}
 		//nolint
-		if err := wait.Poll(5*time.Second, option.WaitTimeout, func() (bool, error) {
+		if err := wait.Poll(5+time.Second, option.WaitTimeout, func() (done bool, err error) {
```

Comment:
why this change while we're not replacing wait.Poll?

Reply:
typo